src/pip/_internal/commands/completion.py

```diff
@@ -28,17 +29,19 @@
                      COMP_CWORD=$(( cword-1 )) \\
                      PIP_AUTO_COMPLETE=1 $words[1] ) )
         }
-        compctl -K _pip_completion pip
+        compctl -K _pip_completion %(prog)s
     """,
     'fish': """
         function __fish_complete_pip
             set -lx COMP_WORDS (commandline -o) ""
-            set -lx COMP_CWORD {cword}
+            set -lx COMP_CWORD ( \\
```

Comment:
Pointing out that this line changed and I've not verified that this change works; I'm pretty sure it would but yeah.

Reply:
I have verified that this still works (I also learned that fish shell is pretty nice!). I changed this line because the `.format` was unnecessary, but these lines still need to be < 80 characters.

Reply:
Yeah. I'd added the `.format` as a hack for making the lines < 80 characters in #4509. I intended to come around to it whenever I would have found the time to fiddle around with `fish` -- a duration long enough to do it justice anyway. :P

> I also learned that fish shell is pretty nice!

It is indeed.